.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "third_party/iree"]
+	path = third_party/iree
+	url = https://github.com/nicolasvasilache/iree.git
```

Comment:
you could take the real iree directly, I was just using my fork because I was maintaining the proper hash there.